jepsen/scalardb/src/scalardb/runner.clj

```diff
@@ -7,11 +7,13 @@
              [core    :as cassandra]
              [runner  :as car]
              [nemesis :as can]]
-            [scalardb.transfer :as transfer]))
+            [scalardb.transfer]
+            [scalardb.transfer_append]))
 
 (def tests
   "A map of test names to test constructors."
-  {"transfer"   transfer/transfer-test})
+  {"transfer"        scalardb.transfer/transfer-test
+   "transfer_append" scalardb.transfer_append/transfer-append-test})
```

Comment:
Just a question. Is there any convention or rule in Clojure or in this file for using `-` and `_` ?

Reply:
As far as I can tell, variables and functions should use "kebab-case" and file names must use "_". I forgot the reason for the latter, but something to do with Java interoperability.

In any case, the test name probably should be `transfer-append` and I have changed that. Thank you!